crates/router/src/connector/multisafepay/transformers.rs

```diff
@@ -237,8 +256,10 @@ impl TryFrom<&types::PaymentsAuthorizeRouterData> for MultisafepayPaymentsReques
         let description = item.get_description()?;
         let payment_options = PaymentOptions {
             notification_url: None,
-            redirect_url: item.request.router_return_url.clone(),
-            cancel_url: None,
+            redirect_url: Some(item.request.get_router_return_url()?)
+                .map(|return_url| format!("{return_url}?status=success")),
+            cancel_url: Some(item.request.get_router_return_url()?)
+                .map(|return_url| format!("{return_url}?status=failure")),
```

Comment:
Since we are expecting router_return _url as mandatory field, can we remove Option from redirect_url & cancel_url field?

Reply:
yes we can remove the option